src/Microsoft.Extensions.Localization.Abstractions/project.json

```diff
@@ -1,6 +1,9 @@
 {
   "version": "1.0.0-*",
-  "description": "Abstractions of application localization services.",
+  "description": "Abstractions of application localization services.\r\nCommonly used types:\r\nMicrosoft.Extensions.Localization.IStringLocalizer\r\nMicrosoft.Extensions.Localization.IStringLocalizer<T>",
```

Comment:
It's something I've been doing when there are some especially interesting types in a package. CoreFX team has been doing this and it seems useful to me. It's not specific to abstractions packages.